src/controller.rs

```diff
@@ -95,10 +95,11 @@ impl Controller for FoxBox {
         let services = self.services.lock().unwrap();
         match services.get(&id) {
             None => {
-                let mut response = Response::with(format!("No Such Service: {}", id));
+                let mut response = Response::with(format!(
+                    "{{\"result\": \"error\", \"details\": \"No Such Service: {}\"}}", id));
```

Comment:
Move to the new json macros, and do something like : { error: "NoSuchService", id: id }